### ISSUE TITLE
[bug]: Paradoxical bug with using withUt or the style itself

### ISSUE THREAD

**Issue by cliffordkleinsr** [pingdotgg/uploadthing#975]
### Provide environment information

```bash
System:
    OS: Windows 11 Pro 23H2
    CPU: AMD Ryzen 7 5700G with Radeon Graphics @ 3.80 GHz
    Memory: 16.0 GB (15.9 GB usable)
    GPU: NVIDIA GTX RTX 4080

  Binaries:
    Node: v20.15.0
    Bun: 1.1.26 
    npm: 10.7.0 

  Browsers:
    Brave Browser: 119.1.60.110
    Chrome: 128.0.6613.139 (Official Build) (64-bit)
    Edge: 128.0.2739.67
    Safari: 17.6
  npm packages:
    "@uploadthing/svelte": "^7.0.2"
    "typescript": "^5.0.0",
    "vite": "^5.0.3"
    "svelte": "^4.2.7",
    "@sveltejs/kit": "^2.5.26",
    "uploadthing": "^7.0.2",
```

### Describe the bug

Using `withUt` breaks the upload button whereas using the `@uploadthing/svelte/styles.css` stylesheet by itself without including the `withUT` utility breaks the navbar.

`WithUT` does not show any loading state when I begin uploading as shown below:
![image](https://github.com/user-attachments/assets/81d212cd-d906-46e4-b6a4-eba16e4379ef)

`Using the @uploadthing/svelte/styles.css  stylesheet import before my `global.css` breaks my navbar as shown below`
![image](https://github.com/user-attachments/assets/eacde586-c2a7-4336-88a0-cb9e828c9106)
which should look like:
![image](https://github.com/user-attachments/assets/e10cb009-f4be-4d32-9ee3-e8af9f042d4d)

***Note*** Im only using withUt  and the style by itself when testing this behavior

### Link to reproduction

N/A

### To reproduce

Add the  `withUT` utility or the styles to a fresh Sveltekit installation with tailwind installed. **But Not Both**!

### Additional information

I have tried clearing my `bun.lockb` and `node-modules` directory and reinstalling the packages. None resolves the issue specifically on deployment to either Vercel or my personal VPS. The stylesheet inclusion works on the dev server however breaks the styles and the navbar on deployment

### 👨‍👧‍👦 Contributing

- [X] 🙋‍♂️ Yes, I'd be down to file a PR fixing this bug!

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by juliusmarminge** [comment]
is your repo a monorepo? if you log the outputted tailwind config, do any uploadthing paths get added to content?

```ts
const config = withUt({
  // your config
})

console.log(config) // does this have `content: ["/path/to/node_modules/@uploadthing/..."]`

export default config
```

**Comment by cliffordkleinsr** [comment]
This is what the console logs:

```ts
{
  darkMode: [ 'class' ],
  content: [ './src/**/*.{html,js,svelte,ts}' ],
  safelist: [ 'dark' ],
  theme: {
    container: { center: true, padding: '2rem', screens: [Object] },
    extend: { colors: [Object], borderRadius: [Object], fontFamily: [Object] }
  },
  plugins: [
    [Function: optionsFunction] {
      __isOptionsFunction: true,
      __pluginFunction: [Function (anonymous)],
      __configFunction: [Function (anonymous)]
    },
    { handler: [Function (anonymous)], config: undefined }
  ]
}
```

**Comment by juliusmarminge** [comment]
> Is your repo a monorepo? What PM are you using?

It's hard to auto-detect the paths in all scenarios. Sometimes `require.resolve("@uploadthing/svelte")` doesn't resolve to anything resulting in no content src being added. In these cases you can do it yourself with a hardcoded path.

**Comment by cliffordkleinsr** [comment]
> monorepo

Yes, it's a mono repo. I'm using vite

**Comment by juliusmarminge** [comment]
Are you able to provide a reproduction? Without that, the only advise I can give is to manually add the content paths

```ts
export default withUt({
  // your config
  content: [
    './src/**/*.{html,js,svelte,ts}',
    './node_modules/@uploadthing/svelte/dist/**',
    '../../node_modules/@uploadthing/svelte/dist/**', // or .. back to root, depending on where your package manager keeps the uploadthing distribution
  ]
})
```

**Comment by cliffordkleinsr** [comment]
Doing it manually works. repro would be hard as its a very large project

**Comment by juliusmarminge** [comment]
No worries I managed to reproduce it in the solid example: https://github.com/pingdotgg/uploadthing/pull/980/commits/1d23a77fc483ef5e1d11a344609ee9932fbe00a4#diff-c91c28746f5b2978ef621d788f91af58b065126119189296cd628141787aa8cd

Guess it's something odd with Vite cause if i `require.resolve("@uploadthing/solid")` in the tailwind config file it finds the path, but the same line of code executed from `uploadthing/tw` doesn't resolve the path...

Will investigate further, but hardcoded paths is a valid workaround for now

**Comment by cliffordkleinsr** [comment]
Many thanks again @juliusmarminge

**Comment by juliusmarminge** [comment]
Fixed in #987. Will be released in the next patch update
